roles/container-engine/cri-o/tasks/main.yaml

```diff
@@ -90,6 +90,20 @@
     remote_src: true
   notify: Restart crio
 
+- name: Cri-o | configure crio to use kube reserved cgroups
+  ansible.builtin.copy:
+    dest: /etc/systemd/system/crio.service.d/00-slice.conf
+    owner: root
+    group: root
+    mode: '0644'
+    content: |
+      [Service]
+      Slice={{ kube_reserved_cgroups_for_service_slice }}
+  notify: Restart crio
+  when:
+    - kube_reserved is defined and kube_reserved is true
+    - kube_reserved_cgroups_for_service_slice is defined
```

Comment:
I don't really like this `is defined` dance when we have role defaults, but given how our roles are layed out right now there is no way around it for now, unfortunately.